_posts/2024-01-03-opensearch-performance-improvements.md

```diff
@@ -0,0 +1,1151 @@
+        <td>281.62</td>
+        <td>73</td>
+        <td>79.53</td>
+        <td>-9%</td>
+        <td>72%</td>
+    </tr>
+</table>
+
+**`pmc` workload results:** The following table illustrates a benchmark comparison for the `pmc` workload between Elasticsearch 7.10.2, OpenSearch 2.3, and OpenSearch 2.11. It includes the 90th percentile of `took` time latency measurements for each (p90) and the observed percentage improvements.
+
+<table>
+    <tr>
+        <th>Operations</th>
+        <th>ES 7.10.2 (p90_value)</th>
+        <th>OS 2.3.0 (p90_value)</th>
+        <th>OS 2.11.0 (p90_value)</th>
+        <th>OS 2.11.0 improvement (vs. OS 2.3.0)</th>
+        <th>OS 2.11.0 improvement (vs. ES 7.10.2)</th>
+    </tr>
+    <tr>
+        <td><code>articles_monthly_agg_cached</code></td>
+        <td>3.55</td>
+        <td>3</td>
+        <td>3.71</td>
+        <td>-24%</td>
+        <td>-5%</td>
+    </tr>
+    <tr>
+        <td><code>articles_monthly_agg_uncached</code></td>
+        <td>12</td>
+        <td>12</td>
+        <td>12.43</td>
+        <td>-4%</td>
+        <td>-4%</td>
+    </tr>
+    <tr class="light-green-clr">
+        <td><code>default</code></td>
+        <td>9</td>
+        <td>6</td>
+        <td>6.79</td>
+        <td>-13%</td>
+        <td>25%</td>
+    </tr>
+    <tr class="light-green-clr">
+        <td><code>phrase</code></td>
+        <td>8.18</td>
+        <td>7</td>
+        <td>7.14</td>
+        <td>-2%</td>
+        <td>13%</td>
+    </tr>
+    <tr class="light-green-clr">
+        <td><code>scroll</code></td>
+        <td>755.18</td>
+        <td>593</td>
+        <td>642.79</td>
+        <td>-8%</td>
+        <td>15%</td>
+    </tr>
+    <tr class="light-green-clr">
+        <td><code>term</code></td>
+        <td>9</td>
+        <td>7</td>
+        <td>7.14</td>
+        <td>-2%</td>
+        <td>21%</td>
+    </tr>
+</table>
+
+## Elevating performance with new features available in OpenSearch 2.11
+
+The following sections present OpenSearch 2.11 features that improve performance.
+
+### LogByteSize merge policy
+
+In the realm of log analytics, the Tiered merge policy has been a cornerstone for efficient shard merges. In OpenSearch 2.11 we introduced the LogByteSize merge policy. This new approach consistently merges adjacent segments, proving especially advantageous for time-series data characterized by timestamp sorting and minimal timestamp overlap between segments.
+
+The following are the key findings from this exercise.
+
+* Timestamp queries with ascending sort had over 75% improvement. This transformation is attributable to the impactful contribution of enhancement [#9241](https://github.com/opensearch-project/OpenSearch/issues/9241).
+* About 40% enhancement in descending sort timestamp queries, surpassing tiered merge policy.
+* Use cases around ascending and descending sort with_after_timestamp saw regression which is a known case for smaller workload with this merge policy.
```

Comment:
This reads as though either "a" or "the" should precede _after_.

Reply:
Changed to "workloads"